packages/lab/src/radio-button/RadioButtonIcon.css

```diff
@@ -20,18 +20,25 @@
   --radioButton-icon-fill-checked: var(--salt-selectable-borderColor-selected);
   --radioButton-icon-fill-checked-disabled: var(--salt-selectable-borderColor-selectedDisabled);
 
+  --radioButton-icon-fill-error: var(--salt-selectable-borderColor-error);
+  --radioButton-icon-fill-error-disabled: var(--salt-selectable-borderColor-selectedDisabled);
+
   --radioButton-icon-strokeColor: var(--salt-selectable-borderColor);
   --radioButton-icon-strokeColor-disabled: var(--salt-selectable-borderColor-disabled);
   --radioButton-icon-strokeColor-hover: var(--salt-selectable-borderColor-hover);
   --radioButton-icon-strokeColor-checked: var(--salt-selectable-borderColor-selected);
   --radioButton-icon-strokeColor-checked-disabled: var(--salt-selectable-borderColor-selectedDisabled);
+  --radioButton-icon-strokeColor-error: var(--salt-selectable-borderColor-error);
+  --radioButton-icon-strokeColor-error-disabled: var(--salt-selectable-borderColor-selectedDisabled);
 
   --radioButton-icon-strokeWidth: var(--salt-size-border);
   --radioButton-icon-strokeWidth-hover: var(--salt-size-border);
```

Comment:
`--radioButton-icon-strokeWidth-hover` isn't used. I think these other strokeWidth tokens also can be deleted and the characteristic used directly instead, since they're only used once it's unnecessary.
+ for the strokeColors above, i see some of them are used in :hover selectors etc or on disabled, but if there's any way we could clean this file up that would be good

Reply:
So the strokeColors should not be used in hover or deleted states?

Reply:
they should use the correct characteristics, but the default, hover, and disabled tokens are only used once, there's no need for the local token to be defined. only the checked token is used twice, and i wonder if there's any way we could merge those selectors anyway - one is 
/* Styles applied to RadioButtonIcon border if `checked={true}` */
the other is
/* Styles applied to RadioButtonIcon border on hover if `checked={true}` */
so if we personally don't change it on hover, maybe we just make it so that they have to write a more specific selector if they want to, and we can get rid of the separation of classes...? idk. regardless, i don't think we need to define the checked local token anyway, it would be fine to just use the characteristic directly if it's just twice

Reply:
Still waiting from design to supply the hover styles. This PR can get merged without those changes. Will update once I get the new values. 
I've cleaned up the vars though. Thanks